src/docs/asciidoc/wan/statistics.adoc

```diff
@@ -0,0 +1,17 @@
+[[wr-stats]]
+=== Statistics
```

Comment:
This section is for synchronization only, and we can do two things with it. Either keep as is and move under the WAN sync section or making it a general monitoring/statistics section and adding other statistics we collect and record as metrics or report via the WAN diagnostics plugin. 

I'm not sure though we want to take the latter way since it will immediately render a lot of work with commenting all the metrics and keeping it in sync between in the code and the manual, which is a bit against the simplicity goal of the metrics work we did. Also, I think the metrics are mostly self-explaining. If not, than it is a very internal metric :smile: 

Therefore, I'd rather move this section under the sync section and I'd call it something like `WAN synchronization statistics`. Also, we need to add that since 4.0 we log the synchronization statistics per member. 

FULL sync:
```
Synchronization statistics:
	 Synchronization UUID: 8af2f9e7-3f9f-4c31-b594-47c421bfb33c
	 Duration: 0 secs
	 Total records synchronized: 448
	 Total partitions synchronized: 5
```
Delta sync:
```
Merkle synchronization statistics:
	 Synchronization UUID: f49a25ba-dc57-4547-817b-bea67ff7f0fe
	 Duration: 0 secs
	 Total records synchronized: 528
	 Total partitions synchronized: 6
	 Total Merkle tree nodes synchronized: 178
	 Average records per Merkle tree node: 2.97
	 StdDev of records per Merkle tree node: 1.55
	 Minimum records per Merkle tree node: 1
	 Maximum records per Merkle tree node: 7
```